projects/win32/f3d.bundle.cmake

```diff
@@ -67,6 +67,13 @@ install(
 
 ## NSIS Generator specific
 if (cpack_generator MATCHES "NSIS")
+
+  # NSIS Resources
+  file(COPY
+      "${superbuild_source_directory}/resources/logo.ico"
```

Comment:
can't we copy the ones located in f3d source?

Reply:
we have no way to access the source file of a project apart from guessing it from the install location, which would be a very bad idea imo.

Reply:
Ok. I think `logotype64.bmp` can be removed from f3d then since it's only used for NSIS.

Reply:
done in f3d-app/f3d#700